inc/features/blocks/namespace.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Experience Block functions.
+ *
+ * @package altis-experiments
+ */
+
+namespace Altis\Experiments\Features\Blocks;
+
+/**
+ * Include and set up Experience Blocks.
+ */
+function setup() {
+	require_once __DIR__ . '/personalization/register.php';
+	require_once __DIR__ . '/personalization-variant/register.php';
```

Comment:
I think these should ideally be in the main plugin.php; if this is a pattern already in the other "features", that's OK, but we should switch in a future PR.